products/knownprojects/sql/create_kpdb.sql

```diff
@@ -32,44 +33,55 @@ WITH
 net_units_details AS (
     SELECT
         *,
-        NULLIF(units_net, 0) IS NOT NULL AS has_future_units,
-        within_5_years + from_5_to_10_years + after_10_years AS net_units_sum,
+        CASE
+            WHEN
+                status = 'DOB 5. Completed Construction'
+                THEN units_net
+            ELSE 0
+        END AS completed_units,
         units_net - (within_5_years + from_5_to_10_years + after_10_years) AS net_units_diff
     FROM _kpdb_combined_and_deduped
 ),
 
 future_units_details AS (
     SELECT
         *,
+        NULLIF(units_net, 0) IS NOT NULL AND completed_units = 0 AS has_future_units,
         CASE
             WHEN
-                has_future_units AND NOT has_project_phasing
+                NOT has_project_phasing AND completed_units = 0
```

Comment:
just clarifying this is intended as `(NOT has_project_phasing) AND completed_units = 0` since that is the way postgres interprets it

Reply:
It seems like it just checking. Also - what's the reason to use logic on this unit value rather than its status?

Reply:
yes to ur first comment, think it's worth adding the parentheses?

Reply:
Nope!

Reply:
logic on the phased unit values seems more accurate and easier to understand than the status

the status values do determine how units get allocated to the the different phases (`within_5_years`, `from_5_to_10_years`, `after_10_years`). I added the table `summary_record_phasing` to show record counts by `source + status`

![Screenshot 2024-07-10 at 2 52 53 PM](https://github.com/NYCPlanning/data-engineering/assets/7444289/4bd6add0-492b-4b6e-8660-c0e5f9b3d5de)